all/pom.xml

```diff
@@ -368,6 +368,7 @@
         <dependency>
             <groupId>com.alibaba</groupId>
             <artifactId>druid</artifactId>
+            <scope>provided</scope>
```

Comment:
seata强依赖druid sqlparser做sql的解析，你为什么要将其scope改为provided，如果你有依赖冲突，应该解决你的依赖冲突问题，而不是提pr到社区粗暴的解决
Seata strong dependency druid sqlparser does sql parsing, why do you want to change its scope to provided, if you have dependency conflicts, you should solve your dependency conflicts instead of mentioning pr to the community for rough resolution

Reply:
`seata-sqlparser-druid`中的`druid`依赖也是`provided`

Reply:
https://github.com/apache/incubator-seata/blob/6011c68f58b9bdc5d14f259b39ea49572c9c6f32/sqlparser/seata-sqlparser-druid/pom.xml#L42-L42

Reply:
Because seata-all strong dependency, it is allowed to provided